platform/conf/conf.go

```diff
@@ -266,11 +282,18 @@ func (c *Conf) copyKeysIgnitionV1(keys []*agent.Key) {
 }
 
 func (c *Conf) copyKeysIgnitionV2(keys []*agent.Key) {
-	var keyObjs []v2types.SSHAuthorizedKey
+	c.ignitionV2.Passwd.Users = append(c.ignitionV2.Passwd.Users, v2types.User{
```

Comment:
I'm fine punting this to the future since, as mentioned, this doesn't break anything in practice and isn't a new issue introduced in this pr.